scripts/custom/drupal-install-site-4-enable-migration.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Please add enabling of this module after `echo "  > Enable development module."` in scripts/custom/drupal-install-site-3...` and remove this file.